### DISCUSSION TITLE
Instant arm with pai

### DISCUSSION THREAD

**Discussion by herebcs** [ParadoxAlarmInterface/pai#486] [Q&A]
Hello :)

The BlueEye app can arm a partition instantly (skipping the delay), if you select ARM twice. Can you do that with PAI? Sending ARM twice with automation does not seem to work...

I have SP7000+ integrated with Home Assistant running on Raspberry PI 4 via 307USB - works perfectly otherwise.

thanks!

**Comment by yozik04** [comment]
Please share what hardware you have.

**Reply by herebcs** [reply]
Sure. Just added to the original post. Thanks!
